docs/src/content/docs/reference/main/stdlib/other.adoc

```diff
@@ -0,0 +1,28 @@
+---
+title: Other Functions
+---
+
+== Length
+
+We offer a generic function to calculate the length/cardinality of elements. 
+
+[source,clojure]
+----
+(length <expr>)
+----
+
+
+[source,sql]
+----
+LENGTH(<expr>)
+----
+
+Where `<expr>` is one of the following:
+
+* A **string** - returns the number of utf8 characters in the string
+** Acts as an alias for the `CHAR_LENGTH` function - see in link:string[String Functions]
+* A **varbinary** - returns the number of bytes in the varbinary
```

Comment:
Usage of `varbinary` - seems to have leaked from arrow datatypes - (ie, https://docs.xtdb.com/reference/main/data-types.html#_scalar_types)